src/data/navigation.ts

```diff
@@ -26,6 +26,23 @@ export const navigation: NavigationItem[] = [
 	{
 		separator: "Code Intelligence",
 		topics: [
+			{
```

Comment:
rather than a separate top-level nav item while also having Cody, it may make more sense to make this a subnav under cody capabilities similar to agentic chat. We can call it "Intent Detection", call out search vs chat in the overview, and have "Search Query Types" as a subsection
when we fully transition the docs away from cody we can reorganize
with omnibox not being a top level navigation name anymore, i'm not sure i like have it at the top level here

Reply:
see https://sourcegraph.slack.com/archives/C0814PENH8R/p1737477843177849